### DISCUSSION TITLE
Nice to have access to the source

### DISCUSSION THREAD

**Discussion by Andre2211** [community/community#122481] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hi,
thanks for supplying the bdtoolset source here.
If you could supply me with the latest sources up to 4.91 for home usage, that would be smashing.
Is it possible?
Greetings,
André

**Comment by ebndev** [comment]
Hi @Andre2211,

Thanks for being a part of the GitHub Community, we're glad you're here!

The Profile category is for conversations related to your GitHub profile page. I have moved this post to General, however, it looks like you're looking for support from [bgtoolset](https://github.com/xcibe95x/bgtoolset). You might want to try asking for help somewhere that focuses on this project, such as opening an issue in the project's repository or seeing if they have a support page.

We want to make sure you’re getting the best support you can and that your message goes where you intended, but this may not be the right place.

Best of luck!

**Comment by ebndev** [comment]
Hi again @Andre2211!

If you don't mind, would you mind letting us know how you initially ended up in the [Profile category](https://github.com/orgs/community/discussions/categories/profile?discussions_q=is%3Aopen+category%3AProfile+) when creating your discussion and providing the link to the project's repository? We are trying to investigate if there is a bug or link that may be re-directing users from the project's Discussions board, if that’s where you intended to post.

Any insight would be helpful, thanks!

**Reply by ebndev** [reply]
Never mind, found the issue! Thanks again for your time and being a part of the GitHub Community 😄
